docs/dev-docs.md

```diff
@@ -12,6 +14,19 @@ Now link the plugin to your QGIS profile python, e.g. for MacOS
     ln -s <path-to-mergin-qgis-plugin>/Mergin/ <path-to-QGIS-user-folder>/QGIS3/profiles/default/python/plugins/Mergin
 ```
 
+### On windows 
+
+Download python [client](https://github.com/MerginMaps/python-api-client), install deps and
+link to qgis plugin. You might need admin privileges to run these commands:
+```
+    mklink /J <path-to-mergin-qgis-plugin>\Mergin\mergin <path-to-python-api-client>\mergin
+```
+
+Now link the plugin to your QGIS profile python, e.g. for MacOS
```

Comment:
`, e.g. for MacOS` - copy-paste typo?

Reply:
It is, thanks for the catch. We can remove it as it's not a "real" path anyway

```suggestion
Now link the plugin to your QGIS profile python:
```